### PR TITLE
[AAE-3007] - inital commit to run eks cluster

### PR DESCRIPTION
**Pull request by SohelSaiyed**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +29,10 @@ before_install:
     chmod +x terraform-docs && sudo mv terraform-docs /usr/bin/
   - curl -L "$(curl -Ls https://api.github.com/repos/terraform-linters/tflint/releases/latest | grep -o -E "https://.+?_linux_amd64.zip")" > tflint.zip &&
     unzip tflint.zip && rm tflint.zip && sudo mv tflint /usr/bin/
+  - curl -sLo /tmp/terraform.zip https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip
```
Comment (mteodori) [inline .travis.yml:32]: why is this needed? it should have been done by the import of:

- Alfresco/alfresco-process-tools:.travis.terraform_install.yml@master

Comment (SohelSaiyed) [reply .travis.yml:32]: this was failing as the Terraform version installed by default was older than the s3 required version, hence I have to force to update the version

Comment (mteodori) [reply .travis.yml:32]: but on new ones it should not have been a problem
